src/BrightcovePlayer.js

```diff
@@ -88,19 +88,24 @@ class BrightcovePlayer extends Component {
           this.props.onCuePoint &&
           this.props.onCuePoint(event.nativeEvent)
         }
+        onID3Metadata={event => {
+          this.props.onID3Metadata &&
+            this.props.onID3Metadata(event.nativeEvent)
+        }
+        }
```

Comment:
Why are these braces in line? Is the indentation not correct here?

Reply:
They were not necessary and I guess fooled the formatter.  I removed them